common/presenters/assessment-answer-to-tag.js

```diff
@@ -1,11 +1,11 @@
 const { TAG_CATEGORY_WHITELIST } = require('../../config')
 
 module.exports = function assessmentAnswerToTag(hrefPrefix = '') {
-  return function map({ key, title, category }) {
+  return function map({ title, category }) {
```

Comment:
is `map` the best name for this? It made me look for the `lodash` map import. Maybe `getTagDetail` or `buildTagDetail`?